sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -1312,4 +1315,12 @@ private Throwable mapError(Throwable throwable, ServiceBusErrorSource errorSourc
     boolean isConnectionClosed() {
         return this.connectionProcessor.isChannelClosed();
     }
+
+    boolean isManagementNodeLocksClosed() {
```

Comment:
In hindsight, I could have made these dependencies, so we wouldn't have to add these package-private methods to assert behaviour. Next time! :)

Reply:
That would be nice. I got the same feeling when trying to figure out how to write the unit tests. :D